lib/algolia/search_client.rb

```diff
@@ -594,12 +594,182 @@ def pending_mappings?(opts = {})
         @transporter.read(:GET, '/1/clusters/mapping/pending' + handle_params({ getClusters: retrieve_mappings }), {}, request_options)
       end
 
-      #
       # Aliases the pending_mappings? method
       #
       alias_method :has_pending_mappings, :pending_mappings?
 
+      # # # # # # # # # # # # # # # # # # # # #
+      # CUSTOM DICTIONARIES METHODS
+      # # # # # # # # # # # # # # # # # # # # #
+
+      # Save entries for a given dictionary
+      #
+      # @param dictionary [String] dictionary name. Can be either 'stopwords', 'plurals' or 'compounds'
+      # @param dictionary_entries [Array<Hash>] array of dictionary entries
+      # @param clear_existing_dictionary_entries [Boolean] delete the previous custom dictionary entries. default: false
+      # @param opts [Hash] contains extra parameters to send with your query
+      #
+      # @return DictionaryResponse
+      #
+      def save_dictionary_entries(dictionary, dictionary_entries, clear_existing_dictionary_entries = false, opts = {})
+        response = @transporter.write(
+          :POST,
+          path_encode('/1/dictionaries/%s/batch', dictionary),
+          { clearExistingDictionaryEntries: clear_existing_dictionary_entries, requests: chunk('addEntry', dictionary_entries) },
```

Comment:
I don't know how chunk works, but will this result in an object like this? `{ type: 'addEntry', body: dictionary_entries[i] }`?

Reply:
the object will look like this 
`{ action: 'addEntry', body: dictionary_entries[i] }`